src/ui/server/helpers/summary-lists/your-business/map-sic-codes/index.ts

```diff
@@ -0,0 +1,21 @@
+import { generateSicCodesValue } from '../../company-house-summary-list';
+import { DEFAULT } from '../../../../content-strings';
+import { ApplicationExporterSicCodes } from '../../../../../types';
+
+/**
+ * mapSicCodes
+ * maps application sic code response to generate a string of the sic code values
+ * @param {Array<ApplicationExporterSicCodes>} applicationSicCodes
+ * @returns {String} sicCodes
+ */
+const mapSicCodes = (applicationSicCodes?: Array<ApplicationExporterSicCodes>) => {
+  if (!applicationSicCodes || !applicationSicCodes.length) {
```

Comment:
Can use `isPopulatedArray` helper here - then only need one unit tests for if array is not populated / is empty array

Reply:
this causes eslint to have fun